src/main/java/com/snowflake/kafka/connector/internal/SFLogger.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2022 Snowflake Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.snowflake.kafka.connector.internal;
+
+/** This class provides an easy way for non-static inheritors to log information. */
+public class SFLogger {
+  private final LoggerHandler logger;
+
+  /**
+   * Constructs an SFLogger instance for the given class.
+   *
+   * @param clazz the class for which the logger is intended
+   */
+  public SFLogger(Class<?> clazz) {
+    this.logger = new LoggerHandler(clazz.getName());
+  }
+
+  /**
+   * Logs an info-level message.
+   *
+   * @param format the message format
+   */
+  public void LOG_INFO_MSG(String format) {
```

Comment:
_MSG looks redundant, could we remove it on all functions?

Reply:
Also LOG_, should we just do INFO, WARN, etc.

Reply:
makes sense for both!